tools/build_pak_files.py

```diff
@@ -57,21 +51,46 @@
     # once we know the platform, set our cooked dir
     unreal_project_cooked_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform + "NoEditor"))
 
-    if spear.path_exists(unreal_project_content_shared_dir):
-        print(f"[SPEAR | build_pak_files.py] File or directory or symlink exists, removing: {unreal_project_content_shared_dir}")
-        spear.remove_path(unreal_project_content_shared_dir)
+    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
```

Comment:
Remove this block because we don't use it here.

```
    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
    # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
    unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
    unreal_project_content_scenes_dir = os.path.join(unreal_project_content_dir, "Scenes")
```

Reply:
We still need `unreal_project_content_scenes_dir`. It's being used inside the `for scene_id in scene_ids:` loop.

Reply:
Ah ok. see my comment below. Let's move this further up so it's logically grouped with `unreal_project_cooked_dir`.

```
    # Once we know the platform, we can set our cooked dir.
    unreal_project_cooked_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform + "NoEditor"))

    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
    # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
    unreal_project_content_scenes_dir = os.path.join(unreal_project_content_dir, "Scenes")
    assert os.path.exists(unreal_project_content_scenes_dir)
```